hedera-node/hedera-app/src/test/java/com/hedera/node/app/state/merkle/MerkleSchemaRegistryTest.java

```diff
@@ -212,9 +218,17 @@ class MigrationTest {
         private MerkleStateRoot merkleTree;
         private SemanticVersion[] versions;
 
+        @TempDir
+        private Path tempDir;
+
         @BeforeEach
         void setUp() {
 
+            final Configuration configuration = new TestConfigBuilder()
+                    .withValue(FileSystemManagerConfig_.TMP_DIR, tempDir.toString())
+                    .getOrCreateConfig();
+            ConfigurationHolder.getInstance().setConfiguration(configuration);
```

Comment:
Is there a way to avoid using `ConfigurationHolder` here (as I tried to get rid of such usages in #15562)?

Reply:
Yes, removed it.